src/sagemaker/model_monitor/clarify_model_monitoring.py

```
@@ -500,14 +524,15 @@ def suggest_baseline(
     # noinspection PyMethodOverriding
     def create_monitoring_schedule(
         self,
-        endpoint_input,
         ground_truth_input,
+        endpoint_input=None,
```

we need to validate that either `endpoint_input` or `batch_transform_input` is given, otherwise, we should raise ValueError and fail fast here